html/reaction.html

```diff
@@ -807,114 +812,143 @@
+                  <tr><td>is normalized?</td><td><div class="product_measurement_is_normalized optional_bool"></div></td></tr>
+                  <tr><td>uses authentic standard?</td><td><div class="product_measurement_uses_authentic_standard optional_bool"></div></td></tr>
+                  <tr>
+                    <td>retention time</td>
+                    <td>
+                      <div class="product_measurement_retention_time_value edittext shorttext floattext"></div>
+                      ± <div class="product_measurement_retention_time_precision edittext shorttext floattext"></div>
+                      <div class="product_measurement_retention_time_units selector" data-proto="Time_TimeUnit"></div>
+                    </td>
+                  </tr>
+                  <tr>
+                    <td>wavelength</td>
+                    <td>
+                      <div class="product_measurement_wavelength_value edittext shorttext floattext"></div>
+                      ± <div class="product_measurement_wavelength_precision edittext shorttext floattext"></div>
+                      <div class="product_measurement_wavelength_units selector" data-proto="Wavelength_WavelengthUnit"></div>
+                    </td>
+                  </tr>
+                  <tr><td>mass spec type</td><td><div class="product_measurement_mass_spec_type selector" data-proto="ProductMeasurement_MassSpecMeasurementDetails_MassSpecMeasurementType"></div></td></tr>
+                  <tr><td>mass spec details</td><td><div class="product_measurement_mass_spec_details edittext"></div></td></tr>
+                  <tr><td>TIC minimum m/z</td><td><div class="product_measurement_mass_spec_tic_minimum_mz edittext shorttext floattext"></div></td></tr>
+                  <tr><td>TIC maximum m/z</td><td><div class="product_measurement_mass_spec_tic_maximum_mz edittext shorttext floattext"></div></td></tr>
+                  <tr>
+                    <td>
+                      EIC masses
+                      <span data-toggle="tooltip" data-placement="right" title="Comma-separated list of m/z ratios that were used to calculate the peak area or intensity.">
+                        <span class="fa fa-question-circle" aria-hidden="true"></span>
+                      </span>
+                    </td>
+                    <td><div class="product_measurement_mass_spec_eic_masses edittext shorttext"></div></td></tr>
+                  <tr><td>selectivity type</td><td><div class="product_measurement_selectivity_type selector" data-proto="ProductMeasurement_Selectivity_SelectivityType"></div></td></tr>
+                  <tr><td>selectivity details</td><td><div class="product_measurement_selectivity_details edittext"></div></td></tr>
+                </table>
+                <!-- Shares "#component_template" with ".components" in the input section. -->
+                <div class="product_measurement_authentic_standard components"></div>
```

Comment:
>I also don't think authentic standards should need to have an amount field inherited from the component template (@michaelmaser - agree?)

Yes, agreed.

Reply:
- The heading size is copied from the `Compound` template, which sits at a different level of nesting. Updated to match the nesting for the authentic standard. It might be nice to write a quick function that recursively updates the heading sizes for all the `fieldset` tags so we don't have to specify them manually...
- Hid the amount section of the form for the authentic standard.